generators/server/templates/pom.xml.ejs

```diff
@@ -44,15 +51,14 @@
         <m2e.apt.activation>jdt_apt</m2e.apt.activation>
         <run.addResources>false</run.addResources>
         <!-- These remain empty unless the corresponding profile is active -->
+<%_ if (databaseType === 'sql') { _%>
```

Comment:
Is liquibase not used for other DB types? Does `'sql'` include all flavors like H2, MySQL, and Postgres, but exclude like Mongo?

Reply:
That's correct